hledger/test/cli/query-args.test

```diff
@@ -4,6 +4,10 @@
   a a  1
   b
 
+2010/3/1 y
+  a    1
+  b
+
```

Comment:
Added some parts input to this as I noticed this test would not fail on some faulty implementations where it should have. The faulty implementation would interpret this query as `Or [Acct $ toRegexCI' "a", Acct $ toRegexCI' "a"]`, where it should have been `Acct $ toRegexCI' "a a"`. The test did not catch this as 'a a' is indeed matched by a regex 'a', so I added this extra transaction.